cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -2713,6 +2703,147 @@ public void testGenerateDeploymentWithOAuthWithClientSecret() {
         );
     }
 
+    @Test(expected = InvalidResourceException.class)
+    public void testOAuthValidationMissingValidIssuerUri() {
```

Comment:
You should test this accross all listeners (the method should be shared, so you do not have to repeat every test 3 times, but it would be great to cover all of them).